src/utils/todo/action-functions.ts

```diff
@@ -145,6 +145,16 @@ export const doResumeFiling = (item: TodoItemI): void => {
       params = { 'conversion-id': item.filingId.toString() }
       break
 
+    case FilingTypes.NOTICE_OF_WITHDRAWAL:
+      // navigate to Notice of Withdrawal page of the old dashboard -- only available for staff account
```

Comment:
1. Change "the old dashboard" to "Filings UI" (also in other filings above). Would also be good to rename "goToBusinessDashboard".

2. I don't see a check for staff account. Is the comment relevant here?

Reply:
Thanks for the updates. Resolving.